website/static/js/contribAdder.js

```diff
@@ -306,6 +332,35 @@ var AddContributorViewModel = oop.extend(Paginator, {
         }
         return false;
     },
+    selectAllNodes: function() {
+        //select all nodes to add a contributor to.  THe changed variable is set here for timing between
+        // treebeard and knockout
+        var self = this;
+        var nodesStateLocal = ko.toJS(self.nodesState());
```

Comment:
I'm not too sure what `nodesStateLocal` means. Could this be named `nodesState` instead?